src/javascript/JContent/ContentRoute/ContentLayout/FilesGrid/FileCard/FileCard.jsx

```diff
@@ -116,7 +117,7 @@ export const FileCard = ({
     let rowNumber = Math.floor(index / 2) + 1;
     let columnNumber = (index % 2) + 1;
     let encodedPath = node.path.replace(/[^/]/g, encodeURIComponent);
-    let isPdf = node.children ? node.children.nodes.filter(node => node.mimeType.value === 'application/pdf').length !== 0 : false;
+    // Let isPdf = node.children ? node.children.nodes.filter(node => node.mimeType.value === 'application/pdf').length !== 0 : false;
```

Comment:
I left it in because we might bring this feature back based on some feedback, according to Francois, but I can take it out it will always be in history if we need it.